plugins/solr.yaml

```diff
@@ -0,0 +1,47 @@
+# Plugin Info
+version: 0.0.1
+title: Solr
+description: Log parser for Solr
+parameters:
+  - name: file_path
+    label: Solr Logs Path
+    description: The absolute path to the Solr logs
+    type: string
+    default: "/var/solr/logs/solr.log"
```

Comment:
Can this be type `strings`? In the past we tend to limit ourselves when using string.